### PR TITLE
Feature/configurable directory start

### PR DESCRIPTION
**Pull request by guyellis**
@dominictarr Following on from the previous PR this is what I was thinking. It's backward compatible with existing and allows consumers to provide a function that will resolve where the rc searching starts from.

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -83,8 +83,7 @@ var env = exports.env = function (prefix, env) {
   return obj
 }
 
-var find = exports.find = function () {
-  var rel = path.join.apply(null, [].slice.call(arguments))
```
Comment (guyellis) [inline lib/utils.js]: I didn't understand why this was here?

Comment (guyellis) [inline lib/utils.js]: Windows/Linux compatibility?

**Comment by dominictarr** [comment]
Sorry, as I mentioned in https://github.com/dominictarr/rc/pull/53#issuecomment-112556883

>@guyellis also note that the pitch for this module is "the non-configurable configuration loader", I only add configuration very reluctantly and I'm fairly confidant there is a better way to solve your problem.

If you look through the other pull requests that have made it into this module, you'll see that nothing was ever merged without considerable diliberation...

**Comment by dominictarr** [comment]
closing, since you didn't make a good case for this change.

**Pull request closed without merging**